Source/Core/GeometryPipeline.js

```diff
@@ -2552,7 +2552,7 @@ define([
         var boundingSphere = geometry.boundingSphere;
         if (defined(boundingSphere)) {
             var minX = boundingSphere.center.x - boundingSphere.radius;
-            if (minX > 0 || BoundingSphere.intersectPlane(boundingSphere, Plane.ORIGIN_ZX_PLANE) !== Intersect.INTERSECTING) {
+            if (minX > 0 || BoundingSphere.intersectPlane(boundingSphere, Plane.ORIGIN_ZX_PLANE) !== Intersect.INTERSECTING || boundingSphere.center.equals(Cartesian3.ZERO)) {
```

Comment:
@bagnell can confirm, but doesn't this just hide the problem and not split the geometry?  Won't that break 2D and Columbus View?